### PR TITLE
fix:ZENKO-3178 move delete button into properties tab

### PR DESCRIPTION
**Pull request by romainlancia**
Screenshot from 2021-03-15 15-36-44

### CODE REVIEW

**Review by Cuervino — APPROVED** [review]
LGTM

**Comment by Cuervino** [comment]
@terrydervaux @nicolas2bert Could you please check/approve this PR?

**Review by terrydervaux — APPROVED** [review]
LGTM @lebroz can you fix the conflict and merge the PR. Thanks

File: src/react/account/details/Properties.jsx [hunk 1]
```diff
@@ -2,50 +2,80 @@
 
 import Table, * as T from '../../ui-elements/TableKeyValue';
 import type { Account } from '../../../types/account';
```
Comment (nicolas2bert) [inline src/react/account/details/Properties.jsx:6]: These two pages seem to have the same layout. Do you mind reusing TableContainer and ButtonContainer components? https://github.com/scality/zenko-ui/blob/development/1.0/src/react/databrowser/buckets/details/Overview.jsx

File: src/react/account/AccountHead.jsx [hunk 2]
```diff
@@ -34,17 +30,13 @@ function AccountHead({ account }: Props) {
         dispatch(closeAccountDeleteDialog());
     };
 
-    // TODO: Should we let the user delete accounts that still owns buckets.
     return (
         <Head>
             { !!account && <DeleteConfirmation show={showDelete} cancel={handleDeleteCancel} approve={handleDeleteApprove} titleText={`Are you sure you want to delete account: ${account.userName} ?`}/> }
```
Comment (nicolas2bert) [inline src/react/account/AccountHead.jsx]: Can you move `DeleteConfirmation` component to `/details/Properties.js` page. You won't need to check if the account exists. Here is an exemple: https://github.com/scality/zenko-ui/blob/development/1.0/src/react/databrowser/buckets/details/Overview.jsx#L83

**Review by nicolas2bert — CHANGES_REQUESTED** [review]

File: src/react/account/details/Properties.jsx (hunk 1 above)
Comment (romainlancia) [reply src/react/account/details/Properties.jsx:6]: done

File: src/react/account/AccountHead.jsx (hunk 2 above)
Comment (romainlancia) [reply src/react/account/AccountHead.jsx]: done

**Comment by romainlancia** [comment]
**UPDATE:**

Screenshot from 2021-04-23 11-43-57

**Review by nicolas2bert — APPROVED** [review]

**Pull request merged**